### ISSUE TITLE
Allow for different login name as username

### ISSUE THREAD

**Issue by xbony2** [fastily/jwiki#5]
Hello. I am a user creating a custom bot for my home wiki, the [Official FTB Wiki](https://ftb.gamepedia.com/FTB_Wiki). We are hosted on a wiki farm called gamepedia, and they have the wikis set up [in a strange way in regards to bots](https://help.gamepedia.com/Logging_in_to_third-party_tools). Basically, bots log in with a bot name that is different from their username. For this reason, I request some sort of built-in way to deal with this using jwiki. Thanks for your time ^^

In the mean time I'm using this hacky solution (in Groovy I should note):
```groovy
wiki = new Wiki(null, null, (HttpUrl)HttpUrl.parse("https://ftb.gamepedia.com/api.php"))
WAction.postAction(wiki, "login", false, FL.pMap("lgname", args[1], "lgpassword", args[2], "lgtoken", wiki.getTokens(WQuery.TOKENS_LOGIN, "logintoken")))
wiki.conf.uname = "ESAEBSAD"
wiki.conf.token = wiki.getTokens(WQuery.TOKENS_CSRF, "csrftoken")
def wlField = Wiki.class.getDeclaredField("wl")
wlField.setAccessible(true)
wlField.get(wiki).put(wiki.conf.hostname, wiki)
		
wiki.conf.isBot = wiki.listUserRights(wiki.conf.uname).contains("bot")
```

**Comment by fastily** [comment]
Hi @xbony2,

Thanks for your feedback!

I believe I resolved this issue about a month ago with e4c5ec71312d236dfcb02327c66b41aadf231e17.  This fix will be included with the next version of jwiki (1.6.0), due to be released within the next few weeks.

**Comment by xbony2** [comment]
Huh, cool ^^ thanks
